Task 2/Task 2.1/Task 2.1.2/User.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Task_2._1._2
+{
+    public class User
+    {
+        public string Name { get; set; }
+        public List<Figure> Figures { get; set; }
+
+        public User(string name)
+        {
+            Name = name;
+            Figures = new List<Figure>();
+        }
+
+        public static implicit operator User(string name)
+        {
+            return new User(name) { Name = name, Figures = new List<Figure>() };
```

Comment:
```suggestion
            return new User(name);
```
Все то, что ты написал в инициализаторе, и так написано в конструкторе